src/AtomShell/window.jl

```diff
@@ -154,71 +163,116 @@ flashframe(win::Window, on = true) =
   @dot_ win flashFrame($on)
 
 """
-    progress(win::Window, p=-1)
+    progress!(win::Window, p=-1)
 
 Sets progress value in progress bar. Valid range is [0, 1.0]. Remove progress
 bar when progress < 0; Change to indeterminate mode when progress > 1.
 
 https://github.com/electron/electron/blob/master/docs/api/browser-window.md#winsetprogressbarprogress-options
 """
-progress(win::Window, p = -1) =
-  @dot_ win setProgressBar($p)
+progress!(win::Window, p) = @dot_ win setProgressBar($p)
```

Comment:
```suggestion
progress!(win::Window, p=-1) = @dot_ win setProgressBar($p)
```
Or did you drop the default value on purpose? If so, can you also drop it from the docstring?